tests/functionaltests/Tests/NSURLSession.mm

```diff
@@ -281,6 +281,7 @@ - (void)URLSession:(NSURLSession*)session downloadTask:(NSURLSessionDownloadTask
 public:
     BEGIN_TEST_CLASS(NSURLSessionTests)
     TEST_CLASS_PROPERTY(L"UAP:AppXManifest", L"NSURL.AppxManifest.xml")
+    TEST_CLASS_PROPERTY(L"ignore", L"true")
```

Comment:
Other tests use Ignore and True or TRUE. I don't think caps matter but maybe we could be consistent?